LearningMissionLab/FinancialEvent.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace LearningMissionLab
+{
+    public class FinancialEvent
```

Comment:
Where is your FinancialIfo class? FinancialInfo class should include FinancialEvent objects. This is pretty similar to Mher's implementation of the Schedule and ScheduleEvent classes. Please take a look.